ESP32LapTimer/src/Wireless.cpp

```diff
@@ -0,0 +1,101 @@
+#include <WiFi.h>
+#include <Arduino.h>
+#include <ESPmDNS.h>
+#include <esp_wifi.h>
+#include <DNSServer.h>
+
+#include "TimerWebServer.h"
+#include "settings_eeprom.h"
+#include "targets/target.h"
+
+static const uint16_t DNS_PORT = 53;
+static IPAddress apIP(192, 168, 4, 1);
+static DNSServer dnsServer;
+static bool airplaneMode = false;
+static uint32_t delayTime = 500; // milliseconds
+static uint32_t maxConnectionTime = 60000; // 2 minutes
+
+void InitWifiAP() {
+  WiFi.begin();
+  delay( 500 ); // If not used, somethimes following command fails
+  WiFi.mode( WIFI_AP );
+  uint8_t protocol = getWiFiProtocol() ? (WIFI_PROTOCOL_11B | WIFI_PROTOCOL_11G | WIFI_PROTOCOL_11N) : (WIFI_PROTOCOL_11B);
+  ESP_ERROR_CHECK(esp_wifi_set_protocol(WIFI_IF_AP, protocol));
+  WiFi.softAPConfig(apIP, apIP, IPAddress(255, 255, 255, 0));
+  uint8_t channel = getWiFiChannel();
+  if(channel < 1 || channel > 13) {
+    channel = 1;
+  }
+  log_i("Starting wifi \"" WIFI_AP_NAME "\" on channel ");
+  log_i("%s", channel);
+  log_i(" and mode ");
+  log_i("%s", protocol ? "bgn" : "b");
+  WiFi.softAP(WIFI_AP_NAME, NULL, channel);
+  // if DNSServer is started with "*" for domain name, it will reply with
+  // provided IP to all DNS request
+  dnsServer.setErrorReplyCode(DNSReplyCode::NoError);
+  dnsServer.start(DNS_PORT, "*", apIP);
+}
+
+bool InitWifiClient() {
+  uint32_t timeWaited = 0;
+
+  log_i("Connecting to: %s", WIFI_SSID);
+  WiFi.begin(WIFI_SSID, WIFI_PASSWORD);
+
+  while(WiFi.status() != WL_CONNECTED) {
+    if(timeWaited >= maxConnectionTime) {
+      log_e("Connection timeout");
+      return false;
+    }
+
+    delay(delayTime);
+    timeWaited = timeWaited + delayTime;
+  }
+
+  log_i("WiFi connected");
+  log_i("IP address: %s", WiFi.localIP().toString().c_str());
+
+  if(!MDNS.begin("chorus32")) {
```

Comment:
This can be removed since you removed the other mDNS calls

Reply:
That one i'd actually like to keep since it enables hostname lookup.
E.g you don't need to know what ip the system get's assigned when working in client mode you just go to http://chorus32.local

Reply:
Nice, definetely keep it!
Maybe we should also set the hostname then? 
At least openwrt resolves <hostname>.lan (or any other suffix you specify).
Also makes it easier to identify the timer in the dhcp list

Reply:
fa6e80c